.pre-commit-config.yaml

```diff
@@ -3,3 +3,10 @@ repos:
   rev: v0.0.265
   hooks:
   - id: ruff
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+  - id: check-added-large-files
+  - id: mixed-line-ending
```

Comment:
1. Could you run `pre-commit autoupdate` to update the versions? I think pre-commit-hooks is currently at v4.4.0.
2. I propose adding `check-merge-conflict` and `end-of-file-fixer` as well.

Reply:
Good suggestions, done. The end-of-file fixer adds a fair number of fixes as part of this PR.